_glaredb/sql-functions/read_mongodb.md

```diff
@@ -6,8 +6,12 @@ parent: SQL functions
 
 # `read_mongodb`
 
-Read a MongoDB collection. The collection does not have to be a known data
-source to GlareDB.
+Read data from an arbitrary MongoDB collection. The collection does
+not have to be a known data source to GlareDB.
+
+If you have multiple collections in one MongoDB deployment or are
+regularly using the same MongoDB data source, consider [setting up a
+MongoDB data source](/docs/data-sources/supported/mongodb).
```

Comment:
**Unrelated to this PR**: This paragraphy/tip is really useful. I have a few thoughts, none of which are concrete.

1) Highlighting it as a tip
2) Applying this tip to other similar table functions
3) At a higher level, explaining what GlareDB is and how to use it/guides

This comment isn't meant to block or be suggestive for this PR to merge.